cmd/lotus-chainwatch/processor/messages.go

```diff
@@ -220,16 +221,22 @@ create temp table msgs (like messages excluding constraints) on commit drop;
 		return xerrors.Errorf("prep temp: %w", err)
 	}
 
-	stmt, err := tx.Prepare(`copy msgs (cid, "from", "to", nonce, "value", gas_premium, gas_fee_cap, gas_limit, method, params) from stdin `)
+	stmt, err := tx.Prepare(`copy msgs (cid, "from", "to", size_bytes, nonce, "value", gas_premium, gas_fee_cap, gas_limit, method, params) from stdin `)
 	if err != nil {
 		return err
 	}
 
 	for c, m := range msgs {
+		var msgBytes int
+		if b, err := m.Serialize(); err == nil {
```

Comment:
Nit: We can avoid this error handling by using the `ChainLen()` method: https://github.com/filecoin-project/lotus/blob/884d03f7f1c35661e28ee853c3cab7cfddb2eabc/chain/types/message.go#L78